### PR TITLE
feat(examples): update angular_bazel_architect to version 11

### PR DESCRIPTION
**Pull request by alan-agius4**

### CODE REVIEW

**Review by mattem — APPROVED** [review]
Awesome 🚀 Reminds me we should update the other [Angular example](https://github.com/bazelbuild/rules_nodejs/tree/stable/examples/angular) too

**Comment by alan-agius4** [comment]
Yeah, If I find sometime I'll try to do them in the coming days.

File: examples/angular_bazel_architect/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -4,5 +4,5 @@ import {RouterModule, Routes} from '@angular/router';
 
 const routes: Routes = [];
 
-@NgModule({imports: [RouterModule.forRoot(routes)], exports: [RouterModule]})
+@NgModule({imports: [RouterModule.forRoot(routes, { relativeLinkResolution: 'legacy' })], exports: [RouterModule]})
```
Comment (saxicek) [inline examples/angular_bazel_architect/src/app/app-routing.module.ts:7]: I think this should be reverted as there are no routes affected by the related bug IMO.

Comment (alan-agius4) [reply examples/angular_bazel_architect/src/app/app-routing.module.ts:7]: @saxicek, agreed. Mind opening a PR? thanks.

Comment (saxicek) [reply examples/angular_bazel_architect/src/app/app-routing.module.ts:7]: Ok, opened https://github.com/bazelbuild/rules_nodejs/pull/2530.

**Pull request merged**